cosmoz-default-tree.html

```diff
@@ -1,10 +1,21 @@
 <link rel="import" href="cosmoz-tree.html">
+
+<!--
+Navigator through object with treelike datastructure and default settings.
+
+@demo demo/index.html
+-->
 <script>
 	'use strict';
 
+	Polymer({ is: 'cosmoz-default-tree' });
```

Comment:
The only reason was for the `iron-component-page` to not crash. (demo & docs)
Maybe there is another way but I didn't explore it yet..